src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -170,22 +193,77 @@ class TaskDeploymentService {
     }
 
     /**
```

Comment:
Everything under this line is related to IAM/json parsing, maybe this should be extracted to a different class/component as it's not directly related to task deployment